www/src/components/evaluation-cell.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+
+class EvaluationCell extends Component {
+  render() {
+    const getBackground = num => {
+      switch (num){
+        case `3`: {
+          return `none`
+        }
+        case `2`: {
+          return `linear-gradient(90deg, transparent 50%, #dddddd 50%)`
+        }
+        case `1`: {
+          return `linear-gradient(180deg, transparent 50%, #dddddd 50%), linear-gradient(90deg, transparent 50%, #dddddd 50%)`
+        }
+        case `0`: {
+          return `none`
+        }
+        case ``:
+        case `N/A`: {
+          return `none`
+        }
+      }
+      return `none`
+    }
+    const basicStyling = {
+      width: `30px`,
```

Comment:
Please replace all pixel values with usages of rhythm. This helps keep spacing consistent across the site.